plugins/websocket/include/websocket/Output.h

```diff
@@ -73,7 +73,9 @@ class PLUGIN_API Output final : public ResultsOutput
 	void operator()(const std::string& metadata, const dto::ImageView*) noexcept final
 	{
 		using namespace boost::json;
-		m_client.sendResult(std::move(parse(string_view{metadata.data(), metadata.size()}).as_object()));
+		parser json_parser;
+		json_parser.write(reinterpret_cast<const char*>(metadata.data()), metadata.size());
```

Comment:
I don't see a need for a cast here. Isn't `const char*` already the returned type?